versioned/persist/mongodb/src/main/java/org/projectnessie/versioned/persist/mongodb/MongoDatabaseAdapter.java

```diff
@@ -100,20 +112,52 @@ public void eraseRepo() {
     client.getCommitLog().deleteMany(idPrefixFilter);
     client.getKeyLists().deleteMany(idPrefixFilter);
     client.getRefLog().deleteMany(idPrefixFilter);
+    client.getAttachments().deleteMany(idPrefixFilter);
+    client.getAttachmentKeys().deleteMany(idPrefixFilter);
   }
 
   private Document toId(Hash id) {
+    return toId(id.asString());
+  }
+
+  private Document toId(String id) {
```

Comment:
Do we need this change anymore? It looks like `toId(String)` is called only from `toId(Hash)` :thinking: